src/rpdk/core/data/schema/provider.definition.schema.v1.json

```diff
@@ -4,6 +4,55 @@
     "title": "CloudFormation Resource Provider Definition MetaSchema",
     "description": "This schema validates a CloudFormation resource provider definition.",
     "definitions": {
+	"handlerSchema": {
```

Comment:
Whats the difference between this and one linked in this github action?  Should we be updating that file instead?  Which one is the source?

https://github.com/aws-cloudformation/cloudformation-cli/blob/fcae34743b962079f29b29b4b7cbe89bdd93c883/.github/workflows/schema-updater.yaml#L11

Reply:
I am not sure what is the difference. @ammokhov do you know?

But I do know that to actually support using this, we need to the change to inliner.py.